ceph-salt-formula/salt/ceph-salt/files/registries.conf.j2

```diff
@@ -0,0 +1,29 @@
+# For more information on this configuration file, see containers-registries.conf(5).
+#
+# Registries to search for images that are not fully-qualified.
+# i.e. foobar.com/my_image:latest vs my_image:latest
+[registries.search]
+registries = ["docker.io"]
+
+{% set insecure_registries = pillar['ceph-salt'].get('container', {}).get('insecure_registries', []) %}
+# Registries that do not use TLS when pulling images or uses self-signed
+# certificates.
+[registries.insecure]
+{% if insecure_registries %}
+registries = [
+{% for registry in insecure_registries %}
+  "{{ registry }}",
+{% endfor %}
+]
+{% else %}
+registries = []
+{% endif %}
+
+# Blocked Registries, blocks the `docker daemon` from pulling from the blocked registry.  If you specify
```

Comment:
does ceph-salt use the docker daemon at all? I thought it used `podman`...

Reply:
podman uses the same config file for registries by default.